src/helpers/provider-instance.ts

```diff
@@ -101,6 +102,18 @@ export async function getProviderInstanceForWallet(
           }),
         );
         break;
+      case WalletType.keystone:
+        cache.set(
+          id,
+          new ProviderKeystoneReactNative({
+            cryptoHDKeyCBORHex: wallet.accountId!,
```

Comment:
Maybe there exists a simpler name?

Reply:
fix in next feature, when add **crypto-account** CBOR support, now supported only **crypto-hd**